lib/verify-auth.js

```diff
@@ -3,14 +3,18 @@ const SemanticReleaseError = require('@semantic-release/error');
 
 module.exports = async (logger) => {
   logger.log('Verify authentication for vsce');
-  const { VSCE_TOKEN } = process.env;
+  const { VSCE_TOKEN, VSCE_PAT } = process.env;
```

Comment:
I think we can actually transform VSCE_TOKEN into VSCE_PAT at the very beginning.

Something like

```js
if(!("VSCE_PAT" in process.env) && ("VSCE_TOKEN" in process.env)) {
  logger.log("VSCE_TOKEN is deprecated and may be removed in the next major release, use VSCE_PAT instead.")
  process.env.VSCE_PAT = process.env.VSCE_TOKEN
}

// do the rest using VSCE_PAT only
```